bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -122,297 +109,175 @@ If you did not manually create things in the *.things file, the Netatmo Binding
-| DateMaxHumidityThisMonth| DateTime         | Date when maximum humidity was reached this month        |
-| Humidex             | Number               | Computed Humidex index                                   |
-| HeatIndex           | Number:Temperature   | Computed Heat Index                                      |
-| Dewpoint            | Number:Temperature   | Computed dewpoint temperature                            |
-| DewpointDepression  | Number:Temperature   | Computed dewpoint depression                             |
-| MinTemp             | Number:Temperature   | Minimum temperature on current day                       |
-| MinTempThisWeek     | Number:Temperature   | Minimum temperature this week                            |
-| MinTempThisMonth    | Number:Temperature   | Minimum temperature this month                           |
-| MaxTemp             | Number:Temperature   | Maximum temperature on current day                       |
-| MaxTempThisWeek     | Number:Temperature   | Maximum temperature this week                            |
-| MaxTempThisMonth    | Number:Temperature   | Maximum temperature this month                           |
-| DateMinTemp         | DateTime             | Date when minimum temperature was reached on current day |
-| DateMinTempThisWeek | DateTime             | Date when minimum temperature was reached this week      |
-| DateMinTempThisMonth| DateTime             | Date when minimum temperature was reached this month     |
-| DateMaxTemp         | DateTime             | Date when maximum temperature was reached on current day |
-| DateMaxTempThisWeek | DateTime             | Date when maximum temperature was reached this week      |
-| DateMaxTempThisMonth| DateTime             | Date when maximum temperature was reached this month     |
-| TimeStamp           | DateTime             | Timestamp when data was measured                         |
-| LastMessage         | DateTime             | Last message emitted by the module                       |
-| LowBattery          | Switch               | Low battery                                              |
-| BatteryVP           | Number               | Battery level                                            |
-| RfStatus            | Number               | Signal strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
+| Channel Group       | Channel Id           | Item Type            | Description                                      |
+|---------------------|----------------------|----------------------|--------------------------------------------------|
+| pressure            | value                | Number:Pressure      | Current pressure                                 |
+| pressure            | absolute             | Number:Pressure      | Pressure at sea level                            |
+| pressure            | trend                | String               | Pressure evolution trend over time               |
```

Comment:
I don't think pressure is supported by additional indoor modules.

Reply:
You're right